src/macros.js

```diff
@@ -961,6 +961,29 @@ defineMacro("\\set", "\\bra@set{\\{\\,}{\\mid}{}{\\,\\}}");
 // actuarialangle.dtx
 defineMacro("\\angln", "{\\angl n}");
 
+//////////////////////////////////////////////////////////////////////
+// physics.sty
+
+// This package contains some macros widely useful in physics.
+// https://ctan.math.washington.edu/tex-archive/macros/latex/contrib/physics/physics.pdf
+
+defineMacro("\\abs", "\\lvert{#1}\\rvert");
+defineMacro("\\Abs", "\\Big\\lvert{#1}\\Big\\rvert");
+defineMacro("\\norm", "\\lVert{#1}\\rVert");
+defineMacro("\\Norm", "\\Big\\lVert{#1}\\Big\\rVert");
```

Comment:
I don't see `\Abs` or `\Norm` in `physics.sty`. Also, it seems that `\abs` and `\norm` generally use `\left` and `\right`, with the ability to customize size via a second argument as in `\abs\Big`. I could see skipping the latter feature in a first version, but I don't think the current definitions match any partial feature from `physics.sty`.